spec/services/idv/session_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe Idv::Session, allowed_extra_analytics: [:*] do
+RSpec.describe Idv::Session do
```

Comment:
This a result from mocking the `EnrollmentHelper.schedule_in_person_enrollment` call. The `schedule_in_person_enrollment` method makes a call to log analytics and this is no longer the case because the method is mocked out in the tests. The FakeAnalytics support code was throwing the following error:

```
Unnecessary allowed_extra_analytics on example group RSpec::ExampleGroups::IdvSession
```

So I just removed the `allowed_extra_analytics: [:*]` because it is no longer necessary.

Reply:
Ahhh, got it, thanks!  That all makes sense. 👍🏻